src/integrationTest/java/integration/tests/TimestampTest.java

```diff
@@ -279,7 +278,7 @@ private void compareAllDateTimeResultSetValuesWithPostgres(ResultSet fireboltRes
 		try (Connection postgresConnection = embeddedPostgres.getPostgresDatabase().getConnection();
 				Statement pgStatement = postgresConnection.createStatement()) {
 			if (timezone != null) {
-				pgStatement.execute(String.format("set timezone = '%s'", timezone));
```

Comment:
Please explain why did we have to remove the equals sign?

Reply:
Good catch. Fixed.